scala/scala.bzl

```diff
@@ -16,6 +16,7 @@
 
 
 _scala_filetype = FileType([".scala"])
+_srcjar_filetype = FileType([".srcjar"])
```

Comment:
sure, though we will still need a _srcjar_filetype to properly filter the files given to _compile which are source jars, no?